src/commands/components/push/operations.test.ts

```diff
@@ -798,26 +872,22 @@ describe('operations', () => {
       }));
 
       vi.mocked(upsertComponent).mockImplementation(async (space, component) => {
-        // Verify that the component's schema has been updated with new IDs/UUIDs
-        if (component.name === 'component-tags') {
-          const schema = component.schema as {
-            tab1: { type: 'tab'; display_name: string; keys: string[] };
-            field1: { type: 'bloks'; restrict_type: string; component_tag_whitelist: number[] };
-          };
-          expect(schema.field1.component_tag_whitelist).toEqual([1001, 1002]);
-          expect(component.component_group_uuid).toBe('new-group-b-uuid');
-          expect(component.internal_tag_ids).toEqual(['1001', '1002']);
```

Comment:
Assertions inside of mock implementations do not work. You can test by checking out the next branch and putting any gibberish here in expected values.

Reply:
False positives @angelikatyborska

Reply:
Yes, they don't crash the test suite or anything, so it looks like the assertions are passing, but they don't do anything

Reply:
Thanks for pointing that out, I will revise them then 🙏🏻, they shouldn't block this PR tho, so we are good